maquetado/scss/styles.scss

```diff
@@ -1,6 +1,7 @@
 .section-project {
-  display: flex;
-  flex-direction: column;
+  display: grid;
+  grid-gap: 5px;
```

Comment:
the `react` is too close to the border line, maybe adding a greater value to this gap could make it look better?